core/filter_api.php

```diff
@@ -3508,3 +3473,94 @@ function filter_print_view_type_toggle( $p_url, $p_view_type ) {
 	);
 	echo '</li>';
 }
+
+/**
+ * Returns an array of project ids which are included in the filter.
+ * This array includes all individual projects/subprojects that are in the search scope.
+ * @param array $p_filter         Filter array
+ * @param integer $p_project_id   Project id to use in filtering, if applicable by filter type
+ * @param integer $p_user_id      User id to use as current user when filtering
+ * @return array
+ */
+function filter_get_included_projects( array $p_filter, $p_project_id = null, $p_user_id = null ) {
+	if( null === $p_project_id ) {
+		$t_project_id = helper_get_current_project();
+	} else {
+		$t_project_id = $p_project_id;
+	}
+	if( null === $p_user_id ) {
```

Comment:
As mentioned in https://github.com/mantisbt/mantisbt/pull/1154#discussion_r133370618, should be loose-type `==` not `===` to catch $p_user_id = 0 or false in addition to null.